examples/struct_from_str.rs

```diff
@@ -46,19 +46,37 @@ fn person_mistyped() -> Result<(), EdnError> {
     Ok(())
 }
 
+fn person_overflow() -> Result<(), EdnError> {
+    let overflow_edn_str = "  {:name \"rose\" :age 9002  }  ";
```

Comment:
Doesn't Edn/Clojure ignore whitespaces?

Reply:
This was just a copy/edit from 
https://github.com/naomijub/edn-rs/blob/554d715adad40cb39b85ee49997b6cf9b40df66d/examples/struct_from_str.rs#L19

This is supposed to fail (changed the `Person` struct to be a u8). It was just the easiest place to add a test because of the `impl Deserialize`, but perhaps it's a realistic example (an error for a person aged over 255).